contrib/IECoreAlembic/include/IECoreAlembic/AlembicInput.h

```diff
@@ -49,12 +49,7 @@ namespace IECoreAlembic
 
 IE_CORE_FORWARDDECLARE( AlembicInput )
 
-/// This class provides very high level access to the contents
-/// of an Alembic cache. It deliberately hides all Alembic
-/// data types and provides an interface tailored to the
-/// reading of 3D scene data only. Finer control and lower level
-/// access can be obtained by using a combination of the Alembic
-/// APIs and the FromAlembicConverters.
+/// \todo Remove
```

Comment:
Yep, and IECoreAppleseed. I'll probably hold off on those until we've got the new renderer API a bit more finalised though.